### ISSUE TITLE
OnCellValueChanged is not triggered when we click out of the grid

### ISSUE THREAD

**Issue by manjuhavanur01** [ag-grid/ag-grid#2832]
We are trying to do some cell level validation on "OnCellValueChanged",
Chrome Browser:
The "OnCellValueChanged" will be triggered only when we click on any other cell after editing but doesn't work if the click happens out of the grid.
Edge Browser:
The "OnCellValueChanged" will be triggered only when we click out of the grid but on any other cell after editing.
Please let us know how can we achieve this.

Response Received:
Can you provide a reproducible scenario using one of our examples or your source code in a plunker and we will investigate?
Accepted reproducible scenarios are
A description of the detailed steps to reproduce your behaviour in one of our examples in the docs.
A plunker
If you decide to send us a plunkr, from any example in our website use the plunkr button in there to fork your own code by following the steps below:
Select the framework that is appropriate to you from the drop-down
Open it in plunker. (Use the button plunker in our example)
Add your changes so that the behaviour is reproduced
Save and Freeze the plunker(On the top left corner)
Send us the link to the plunker(You can copy the URL from the browser)
If reporting a bug make sure to state.
Actual behaviour.
Expected behaviour. If possible back this up with our docs/examples if possible

Hi Team,

Please find the following plunkr link with example to reproduce the scenario

**https://embed.plnkr.co/PNUZwICHVaAgxIfR/**

Actual behavior.      [**Microsoft Edge**]

1. Double click on any cell
2. Edit the value and click any other cell (focus out event does not triggers before "**oncellValueChanged**")
3. Edit the value and click outside of the Ag-grid (focus out event will be triggered then goes to "**oncellValueChanged**")

We are calling focus out here because we need to decorate the cell based on the values entered here.

Expected behavior.   [**Microsoft Edge**]

1. Double click on any cell
2. Edit the value and click any other cell (focus out event must trigger then it should fire "**oncellValueChanged**")
3. Edit the value and click outside of the Ag-grid (focus out event must trigger then go to  "**oncellValueChanged**")

**Comment by gauravSng1994** [comment]
The actual prop is **stopEditingWhenCellLosesFocus** not **stopEditingWhenGridLosesFocus**

**Comment by stanislaugh** [comment]
> The actual prop is **stopEditingWhenCellLosesFocus** not **stopEditingWhenGridLosesFocus**

No, it's `stopEditingWhenCellsLoseFocus`, not `stopEditingWhenCellLosesFocus` 😅

https://www.ag-grid.com/react-data-grid/grid-options/#reference-editing-stopEditingWhenCellsLoseFocus

**Comment by S1lander** [comment]
> > The actual prop is **stopEditingWhenCellLosesFocus** not **stopEditingWhenGridLosesFocus**
> 
> No, it's `stopEditingWhenCellsLoseFocus`, not `stopEditingWhenCellLosesFocus` 😅
> 
> https://www.ag-grid.com/react-data-grid/grid-options/#reference-editing-stopEditingWhenCellsLoseFocus

Actually now it's `stopEditingWhenCellsLoseFocus` hahah. I think this should be set to true by default tho. Thanks guys.
